swagger.yaml

```diff
@@ -397,6 +397,12 @@ paths:
                 type: array
                 items:
                   $ref: '#/definitions/UnterlagenFreigabe'
+        - in: query
```

Comment:
Innerhalb der Query finde ich die Callback URL potentiell sehr gefährlich. Da muss das Escaping korrekt sein und die URL darf nicht zu lang sein.
Ich würde es genau wie `empfaenger` sowie `antragsNummer` mit in das Freigabe Objekt schieben.